cPP_Lifecycle.adoc

```diff
@@ -0,0 +1,135 @@
+:iTC-shortname: BIO-iTC
+:iTC-email: isec-itc-bio-info@ipa.go.jp
+:iTC-website: https://biometricitc.github.io/
+:iTC-GitHub: https://github.com/biometricITC/cPP-biometrics
+:iTC-ITname: BIT
+
+== Introduction
+As part of the normal work of the {iTC-shortname}, new iterations of the cPP and SD will be released over time. Ideally, when a new version of the cPP is released, everyone would immediately follow the new version, but of course this is not possible as it takes time to transition to updated requirements.
+
+The purpose of this document is to specify the lifecycle for the cPP versions, providing guidance on when a new version will supersede earlier versions (when the previous version is no longer considered acceptable for use) and how this will be announced to the wider community.
+
+[NOTE]
+====
+While the document talks about cPP revisions, this covers the cPP, SD and any other related documents.
+====
+
+== Revision Classes
+There are three classes of revisions to a cPP that can be produced. The descriptions are not meant to be exhaustive, but to represent the level of changes in the document.
+
+.Revision Classes for a cPP
+[cols=".^1,.^1,.^4",options="header"]
+|===
+
+|Revision Class
+|Version Position
+|Description
+
+|Major
+|**X**.y.zz
+|Full update to the cPP, including requirements changes (addition or removal of requirements)
+
+|Minor
+|x.**Y**.zz
+|Update to the cPP, likely updating existing requirements, but not changing them (i.e. not adding/removing requirements)
+
+|Technical Decision or Errata
+|x.y.**ZZ**
+|Application of Technical Decisions to the cPP (no other changes)
+
+|===
+
+=== Revisions & Branches
+The {itc-shortname} maintains 3 primary branches for the creation of the cPP and all supporting documents. Of these branches, two will be used to maintain the current versions of the documents while the third is used for active work on new updates.
+
+.Branches & the Latest Documents
+[cols=".^1,.^1,.^4",options="header"]
+|===
+
+|Branch
+|Versions
+|Description
+
+|Master/Main
+|**X**.**Y**
+|This is always maintained at the last Major.Minor release and is the "official" published version.
+
+|Interpretation
+|x.y.**ZZ**
+|This branch will be updated for each TD that is published and is a rolling update for all approved TDs between Major.Minor updates.
+
+This branch can be used in place of Major.Minor + TDs if the validating scheme approves all TDs.
+
+|===
+
+The {itc-shortname} may approve errata updates on a periodic basis to officially roll TDs into a published release depending on the number of TDs and the time before the next full update.
+
+== Revision Transitions
+When a new revision will be published, it is immediately available for use. So if version 1.1 is released, it can be immediately used for an evaluation. To ensure a common direction and basis for evaluation within the community, the release of a new cPP revision means that the previous revisions must be retired from active use.
```

Comment:
```suggestion
When a new revision is published, it is immediately available for use. So if version 1.1 is released, it can be immediately used for an evaluation. To ensure a common direction and basis for evaluation within the community, the release of a new cPP revision means that the previous revisions must be retired from active use.
```